SECURITY.md

```diff
@@ -11,7 +11,8 @@ currently being supported with security updates.
 
 | Version  | Supported          | Notes                  |
 | -------- | ------------------ | ---------------------- |
-| 3.x      | :white_check_mark: |                        |
+| 4.x      | :white_check_mark: |                        |
+| 3.x      | :x:                | End-of-life as of 2021 |
```

Comment:
Maybe add the month when 3.x went EOL? Since it was actively developed most of 2021.
```suggestion
| 3.x      | :x:                | End-of-life as of November 2021 |
```

Reply:
Make sense. Thanks @7632